pyoverkiz/client.py

```diff
@@ -244,8 +248,6 @@ async def somfy_tahoma_refresh_token(self, refresh_token: str) -> str:
                 seconds=token["expires_in"] - 5
             )
 
-            return self._access_token
```

Comment:
It was not used, as we can directly use `self._access_token`

Reply:
Indeed! Do we need to check if we have a refresh token, or can this race condition never happen.

Reply:
In our code, we have already a check: https://github.com/iMicknl/python-overkiz-api/blob/0c45f2e6a3ae20b7af77ec8cd976a393a2a3f780/pyoverkiz/client.py#L729

but not directly in this method,  I can raise an error in such case. What do you think?

Reply:
Would be good to raise an error / exception (or call the other function), just to be sure.